trin-state/src/utils.rs

```diff
@@ -18,6 +23,69 @@ const MID: [u8; 77] = [
     49, 57, 57, 54, 56,
 ];
 
+pub fn state_network_derive_content_id(key: &String) -> U256 {
+    // SSZ decode ContentKey
+    let decoded_key = match StateContentKey::from_bytes(&key.as_bytes()) {
+        Ok(k) => k,
+        Err(_) => panic!("Cannot decode key")
```

Comment:
Looks like the derive method should return a `Result` instead. Crashing the program on an invalid content key sounds bad.